### DISCUSSION TITLE
Expand pokemob AI with Minecraft Mobs mechanics

### DISCUSSION THREAD

**Discussion by NelkGames** [Pokecube-Development/Pokecube-Issues-and-Wiki#600] [Ideas]
Here is a list of Pokemons that could expand their AI and gain skills that some minecraft mobs have.

--Dolphin Jump-- Dolphin
Seel,Dewgong,Mantyke,Mantine,Sharpedo,Magikarp,Gyarados,Milotic,
Phione,Manaphy,Popplio,Brionne,Primarina,Arrokuda and Barraskewda

--Avoid Creepers-- Cats
All pokémon that are cats

--Gifts-- Cats
All pokémon that are cats
Delibird

--Pick up items with the mouth-- Fox
Rattata, Raticate, Nickit

--Imitate the sound of nearby mobs-- Parrot
Chatot

--Restore life with iron ingots-- Iron Golem
Gollet, Golurk, Celesteela, Magearna, Beldum, Skarmory

--Climb walls-- Spider
Spinarak, Ariados

--Walking on Lava-- Strider
Heatran, Magmar, Magmortar, Slugma, Magcargo

--Walking on Water-- Strider
Dewpider, Araquanid, Suicune, Keldeo

--Generate Snow-- Snow Golem
Regice, Avalugg, Abomasnow, Glalie

**Comment by Thutmose** [comment]
the creepers one will be trivial, there is already something similar for them avoiding psychic types (which tend to detonate them)

for items in the mouth one, that can probably be done with similar code for rendering the worn items, can have them render their first inventory slot like that
